charts/kong/values.yaml

```diff
@@ -183,8 +183,21 @@ plugins: {}
 # - kong-admin-tls
 secretVolumes: []
 
-# Set runMigrations to run Kong migrations
-runMigrations: true
+# Enable/disable migration jobs, and set annotations for them
+migrations:
+  # Enable initial migrations/"kong migrations bootstrap"
+  # This should be set to false after the initial install
+  init: true
```

Comment:
Is it possible to detect if a rendering process is an install or a delete inside the template?
If so, we can further automate this. If not, NBD.

Reply:
It is! And it handles deleting it automatically too!